static/app/components/stream/group.tsx

```diff
@@ -467,10 +468,10 @@ function BaseGroupRow({
           ) : null}
           {withColumns.includes('assignee') && (
             <AssigneeWrapper narrowGroups={narrowGroups}>
-              <DeprecatedAssigneeSelector
-                id={group.id}
+              <AssigneeSelectorDropdown
+                group={group}
```

Comment:
`<AssigneeSelectorDropdown>` only uses the group.id, group.owners, and group.assignedTo property. Should I just pass these three in as props instead of the entire group?